float64 xPose
float64 yPose
